content/guides/ai-studio/index.md

```diff
@@ -0,0 +1,46 @@
+---
+rank: 0
+related_guides:
+	- box-ai/ai-studio/create-agents
+	- box-ai/ai-tutorials/ask-questions
+	- box-ai/ai-tutorials/generate-text
+---
+
+# Box AI Studio
+
+<Messsage type='caution'>
+Box AI Studio is available only for Enterprise Advance accounts.
+</Message>
+
+[Box AI Studio][ai-studio] allows you to build and manage custom AI agents to
+best suit your business needs. For example, you can create an AI agent
+that acts as a compliance consultant, answering questions regarding customer
+documentation with the FedRAMP Moderate compliance in mind. 
+
+## Box AI Studio capabilities
+
+Currently, you can configure Box AI agent to answer user questions or generate
+text you can use in your documents.
+
+![ai agent capabilities](./images/ai-agent-capabilities.png)
+
+## Supported languages
+
+Box AI studio works in a number of languages including
+English, Japanese, French, Spanish, and many more.
+However, the underlying models are primarily
+trained on English language documents. This means
+that prompts in other languages may return answers
+of lower quality than in English. Tests have shown
+satisfactory results for summarizing, checking grammar
+and spelling, and answering questions, but bear in mind
+that the results may be different than in English.
+
+<Message type="tip">
+
+Switch the language to Japanese to get
+better results for this language.
+
+</Message>
+
```

Comment:
I noticed in Box AI main guide there is [Box AI API in User Activity Report (UAR)](https://developer.box.com/guides/box-ai/#box-ai-api-in-user-activity-report-uar) section, should we also inlcude it here?